securedrop-export/debian/control

```diff
@@ -0,0 +1,17 @@
+Source: securedrop-export
+Section: utils
+Priority: optional
+Maintainer: SecureDrop Team <securedrop@freedom.press>
+Build-Depends: debhelper (>= 9), dh-python, python3-all, python3-setuptools, dh-virtualenv
+Standards-Version: 3.9.8
+Homepage: https://github.com/freedomofpress/securedrop-export
+Vcs-Git: https://github.com/freedomofpress/securedrop-export.git
+X-Python-3-Version: >= 3.5
+
+Package: securedrop-export
+Architecture: all
+Depends: ${python3:Depends}, ${misc:Depends}
```

Comment:
Would it make sense to add the dependencies that are strictly required for this script to work (eg, printer drivers, xpp, cups)?
https://github.com/freedomofpress/securedrop-workstation/blob/503fd6f0930a24ddc92d9cfae4fa502275b3362f/dom0/sd-export-files.sls#L15:L25

Reply:
Sure, although would they be better off in a metapackage? (might be getting into gilding the lily a bit at that stage tho.)